IBX-10133: [Rector] Added plugin to rename icons names (#40)
ibexa/rector: js/README.md

|---|---|---|
| true | `/*@Desc("<p class='ez-not-error--show-modal'>Show message</p> for ez-not-error--show-modal")*/`  | `/*@Desc("<p class='ibexa-not-error--show-modal'>Show message</p> for ez-not-error--show-modal")*/` |
| false | `/*@Desc("<p class='ez-not-error--show-modal'>Show message</p> for ez-not-error--show-modal")*/` |  `/*@Desc("<p class='ibexa-not-error--show-modal'>Show message</p> for ibexa-not-error--show-modal")*/` |

### Rename icons names used in getIconPath method

> This plugin allows to rename any icon name that is used as an argument in getIconPath method.

Reviewer: ```suggestion
This plugin allows you to rename any icon name that is passed as an argument to the `getIconPath` method.
```

**Name:** `ibexa-rename-icons`

**Configuration example:**

> !IMPORTANT by default in this plugin shorthand expression is treated as `exactMatch: true`.

Reviewer: ```suggestion
In this plugin, the `exactMatch` default value is set `true` when using the shorthand expression.
```
